router/http/src/main/scala/io/buoyant/router/http/TracingFilter.scala

```diff
@@ -10,29 +10,31 @@ import com.twitter.finagle.tracing.Trace
 object TracingFilter extends SimpleFilter[Request, Response] {
 
   def apply(req: Request, service: Service[Request, Response]) = {
-    Trace.recordBinary("http.method", req.method.toString)
+    Trace.recordRpc(req.method.toString)
+    // http.uri is used here for consistency with finagle-http's tracing filter
     Trace.recordBinary("http.uri", req.uri)
+    Trace.recordBinary("http.req.method", req.method.toString)
     for (h <- req.host) {
-      Trace.recordBinary("http.host", h)
+      Trace.recordBinary("http.req.host", h)
     }
-    recordMessage(req)
+    recordMessage("req", req)
 
     service(req).onSuccess { rsp =>
-      Trace.recordBinary("http.status", rsp.status.code)
-      recordMessage(rsp)
+      Trace.recordBinary("http.rsp.status", rsp.status.code)
+      recordMessage("rsp", rsp)
     }
   }
 
-  private[this] def recordMessage(msg: Message): Unit = {
-    Trace.recordBinary("http.version", msg.version.toString)
+  private[this] def recordMessage(scope: String, msg: Message): Unit = {
+    Trace.recordBinary(s"http.$scope.version", msg.version.toString)
     for (length <- msg.contentLength) {
-      Trace.recordBinary("http.content-length", length)
+      Trace.recordBinary(s"http.$scope.content-length", length)
     }
     for (t <- msg.contentType) {
-      Trace.recordBinary("http.content-type", t)
+      Trace.recordBinary(s"http.$scope.content-type", t)
     }
     for (te <- msg.headerMap.get("transfer-encoding")) {
-      Trace.recordBinary("http.transfer-encoding", te)
+      Trace.recordBinary(s"http.$scope.transfer-encoding", te)
```

Comment:
tioli: not a _huge_ deal but we could probably do this with fewer allocations by having recordReq and recordRsp methods that use val'd strings (aiui this will have to do new string allocations on each message...)